app/controllers/candidate_interface/section_controller.rb

```diff
@@ -0,0 +1,25 @@
+module CandidateInterface
+  class SectionController < CandidateInterfaceController
+    before_action :set_section_policy
+    before_action :verify_authorized_section, except: %i[show review]
+
+    def show; end
```

Comment:
Or should we disable [the linter](https://www.rubydoc.info/gems/rubocop/0.58.0/RuboCop/Cop/Rails/LexicallyScopedActionFilter) instead? I feel like we'll have a lot of empty methods like this. We could move it into the parent controller, but I feel that would become messy. Thoughts? @DFE-Digital/find-and-apply-devs

Reply:
I disabled previously but I understood from Friday that we want empty methods. I am happy either way, don't have a strong opinion about.

Reply:
You disabled it for those rows, but I'm having the same issue in my PR. So we might as well disabled it globally or add it to the todo.

Reply:
I was planning to merge this as soon is green, but let's talk on slack, I want to understand what you need on your PR.